lxd/daemon.go

```diff
@@ -764,6 +764,12 @@ func (d *Daemon) createCmd(restAPI *mux.Router, version string, c APIEndpoint) {
 			return
 		}
 
+		// Generate an OpenFGA request ID
+		_, err = request.GetCtxValue[*openfga.OpenFGACache](r.Context(), request.CtxOpenFGARequestCache)
+		if err != nil {
+			request.SetCtxValue(r, request.CtxOpenFGARequestCache, openfga.NewOpenFGACache())
+		}
+
```

Comment:
Why get it first and then set it if not present? At this point we can be sure that it has not been set. 

Also could you please move this up to where the username and protocol are being set and use the same pattern. Thanks.

Reply:
But what happens if this is an API call from an other cluster member that already created this cache in the request context? This check would avoid re-creating the cache right?

Reply:
If the request is forwarded by the other member then the cache was in-memory in the other member and is no longer available in the request context.